marketplace/src/main/resources/install/templates/mongodb-directory/nxserver/config/mongodb-directories-config.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.directories.mongodb">
+
+  <implementation class="org.nuxeo.directory.mongodb.MongoDBDirectoryDescriptor"/>
+
+  <require>org.nuxeo.directory.mongodb.MongoDBDirectoryFactory</require>
```

Comment:
You don't need to require `org.nuxeo.directory.mongodb.MongoDBDirectoryFactory` as it's implicitly done given that we use it as an extension point target.